### PR TITLE
Add support for jdbi3

### PR DESCRIPTION
**Pull request by usulkies**
Added a new package for jdbi3 support.
Copied the class and changed code and documentation to match Jdbi3 syntax and semantics.
Tests are copied also to test the new class.

### CODE REVIEW

**Comment by usulkies** [comment]
I'm not sure it's a good idea.
Jdbi 2 and 3 are different artifacts, and upgrading to it is [not so simple](http://jdbi.org/#_upgrading_from_v2_to_v3).
Better to support them both.

File: pom.xml [hunk 1]
```diff
@@ -89,6 +92,13 @@
             <groupId>org.jdbi</groupId>
             <artifactId>jdbi</artifactId>
             <version>${jdbi.version}</version>
+            <scope>provided</scope>
```
Comment (usulkies) [inline pom.xml]: I changed the scope to "provided", assuming the import of jdbi is done explicitly by the user of this library, and not transitively through the import of the opentracing for jdbi.
This way we make sure that users of jdbi v2 will not get the jdbi3 imported to their packages unwillingly, and vice-versa.

File: pom.xml [hunk 2]
```diff
@@ -70,12 +70,15 @@
     </issueManagement>
 
     <properties>
-        <java.version>1.7</java.version>
+        <java.version>1.8</java.version>
```
Comment (usulkies) [inline pom.xml]: jdbi3 requires java 8

**Comment by malafeev** [comment]
thank you @usulkies

**Pull request merged**